### ISSUE TITLE
mislabeled issue as feature

### ISSUE THREAD

**Issue by SKGrimes** [gfidente/pelican-svbhack#26]
You put no ie testing as a `known issue` and not a feature. To close issue please relabel as feature.

**Comment by gfidente** [comment]
pardon me but I am not a native english speaker so I might not understand the issue fully

are you suggesting that 'no IE testing' should actually go in the 'features' section of the README file? isn't it a potential cause of issues instead?

**Comment by HybridAU** [comment]
Yes, I think SKGrimes is making a joke, suggesting that not testing in IE is a good thing as it will discourage people from using that browser.

**Comment by gfidente** [comment]
ah we're in 2023 now and it's probably worth removing that line from the README entirely?
